src/sentry/static/sentry/app/components/alert.tsx

```diff
@@ -65,7 +65,17 @@ const StyledTextBlock = styled(TextBlock)`
   align-self: center;
 `;
 
-const Alert = ({type, icon, iconSize, children, className, ...props}) => {
+type AlertProps = {
+  type?: string;
```

Comment:
We can do better on the type

https://github.com/getsentry/sentry/pull/14703/files#diff-3d15e1f6a6441ac3fdf4f164638bcaefR12

Reply:
yeah, i wasn't able to get this to compile, will give it another shot later.